python/tvm/script/builder/tir/__init__.py

```diff
@@ -31,3 +31,45 @@
 )
 from .prim_func_frame import arg, prim_func
 from .var import Buffer
+
+from tvm.script.tir import (
+    uint8,
+    int8,
+    int16,
+    int32,
+    int64,
+    float16,
+    float32,
+    float64,
+    boolean,
+    Ptr,
+)
+from tvm.script.tir.intrin import comm_reducer
```

Comment:
Let's make sure that we don't depend on the existing `tvm.script.tir` namespace, which is supposed to be deprecated.